src/components/ActionCheckbox/ActionCheckbox.vue

```diff
@@ -46,7 +46,12 @@ export default {
 	props: {
 		id: {
 			type: String,
-			default: 'checkbox-' + GenRandomId()
+			// we have to use computed/data
+			// or else GenRandomId will return the same string
+			// for all the ActionCheckbox instances
+			// most likely that the code gets executed on compile instead
```

Comment:
this code is evaluated when the module is parsed. you could try using a factory function like `default: () => GenRandomId()`

Reply:
Ooooh! You're right, lemme try!